product_tax_multicompany_default/tests/test_product_tax_multicompany.py

```diff
@@ -139,6 +140,16 @@ def test_tax_not_default_set_match(self):
 
     @users("user_12")
     def test_set_multicompany_taxes(self):
+        # If purchase module is installed
+        # add purchase manager group to user_12
+        # to access the supplier_taxes_id field in the product view
+        module = self.env["ir.module.module"].search(
+            [("name", "=", "purchase"), ("state", "=", "installed")]
+        )
+        if module:
+            self.env.ref("purchase.group_purchase_manager").sudo().users = [
+                (4, self.user_12.id)
+            ]
```

Comment:
Thanks for the improvement! 
You can simplify with `raise_if_not_found=True` if you want, but It's ok for me as it is ;)

```suggestion
        try:
            self.env.ref(
                "purchase.group_purchase_manager", 
                raise_if_not_found=True
            ).sudo().users = [
                (4, self.user_12.id)
            ]
        except ValueError:
            pass  # Skipping configuration of purchase module
```

Reply:
It's done. Thanks for the advice.